src/libstd/sys/unix/ext/process.rs

```diff
@@ -67,10 +67,20 @@ pub trait CommandExt {
     /// an error indicating why the exec (or another part of the setup of the
     /// `Command`) failed.
     ///
+    /// `exec` not returning has the same implications as calling
+    /// [`process::exit`] – no destructors on the current stack or any other
+    /// thread’s stack will be run. Therefore, it is recommended to only call
+    /// `exec` at a point where it is fine to not run any destructors. Note,
+    /// that `execvp` syscall independently guarantees that all memory is freed
+    /// and all file descriptors with the `CLOEXEC` option (set by default on all
+    /// file descriptors opened by the standard library) are closed.
```

Comment:
It is not. We always try our best to set CLOEXEC on all filedescriptors in the standard library and the behaviour of `execvp` wrt `CLOEXEC` fds is documented in posix: http://pubs.opengroup.org/onlinepubs/9699919799/functions/execvp.html